### ISSUE TITLE
Wrong warning: The WordPress CRON jobs system is disabled.

### ISSUE THREAD

**Issue by mahnunchik** [Automattic/jetpack#25477]
I've faced with incorrect warning:

```
The WordPress CRON jobs system is disabled. This means the garbage collection system will not work unless you run the CRON system manually.
```

Yep, the variable `DISABLE_WP_CRON` set to `true`. But cron works through system cron https://developer.wordpress.org/plugins/cron/hooking-wp-cron-into-the-system-task-scheduler/#macos-and-linux

How to remove incorrect warning?

**Comment by mahnunchik** [comment]
Any news?

**Comment by d-packs** [comment]
This issue has nothing to do with Super Cache. However if moving it to this component solves it faster I'm game.

**Comment by JeffBurris** [comment]
> This issue has nothing to do with Super Cache. However if moving it to this component solves it faster I'm game.

Well for me it has a bit to do with Super Cache because I never get the error until I install Super Cache on any site. I suppose you're saying all Super Cache is doing is implementing garbage cleanup and exposing an erroneous error inherent in Wordpress?

**Comment by gaswirth** [comment]
This issue only occurs for me when using Super Cache. In fact, the only place the error string appears (`The WordPress CRON jobs system is disabled. This means the garbage collection system will not work unless you run the CRON system manually.`) is in Super Cache's main `wp-cache.php` plugin file. We, too, are using a system CRON setup for performance, and our client thinks there's something wrong with her site, where it's just this erroneous error.

**Comment by d-packs** [comment]
> I suppose you're saying all Super Cache is doing to expose it is implementing garbage cleanup

I'm saying I experienced the issue without having SuperCache installed. I'm not using that plugin at all.

**Comment by JeffBurris** [comment]
why is this being addressed to me, I haven't commented about this other than reporting what happens on my setup. I knpow nothing about how garbage cleanup works and would not claim to. I don't remember subscribing to this thread and amd receiving no other messages...
________________________________

**Comment by jeherve** [comment]
@JeffBurris When you comment on a GitHub issue, you are automatically subscribed to all updates on that issue. If you'd rather not receive updates for each comment here, I would recommend clicking on the "Unsubscribe" button in the sidebar of the issue page.

**Comment by boltronics** [comment]
I don't think we even need to go so far as to have a checkbox in the plugin (although I wouldn't complain if such a thing existed). I think just having the user set another variable similar to `DISABLE_WP_CRON` would be sufficient. eg. having something like`OS_CRON_CONFIGURED` manually set to true by the administrator.

This way, it's a trivial matter of adding a single `if` statement to `wp-cache.php`, which would hopefully allow this issue to be resolved very quickly.

**Comment by madalinignisca** [comment]
Would be wonderful if this warning can be turned off by a constant for all of us which run the cron from the system cron (I use wp-cli on linux system cron for example for this).
